src/ToDo/Presentation/TaskNoteViewModel.cs

```diff
@@ -2,5 +2,23 @@ namespace ToDo.Presentation;
 
 public partial class TaskNoteViewModel
 {
-    
-}
\ No newline at end of file
+	private INavigator Navigator { get; }
```

Comment:
There is no needs to use a property for this, please convert to a simple field

```suggestion
	private readonly INavigator _navigator;
```

Reply:
done